index.js

```diff
@@ -49,32 +49,32 @@ async function onTrack(event, settings) {
         event.properties['$current_url'] = event.properties.url
         delete event.properties.url
     }
-    
+
     if (event.properties && event.properties.utm_name) {
-        event.properties["utm_campaign"] = event.properties.utm_name
+        event.properties['utm_campaign'] = event.properties.utm_name
     }
-  
+
     if (event.properties && event.properties.browser) {
-        event.properties["$browser"] = event.properties.browser
+        event.properties['$browser'] = event.properties.browser
     }
-  
+
     if (event.context && event.context.groupId) {
-        if(!event.properties) {
-          event.properties = {}
+        if (!event.properties) {
+            event.properties = {}
         }
-        event.properties["$groups"] = {"segment_group": event.context.groupId}
+        event.properties['$groups'] = { segment_group: event.context.groupId }
     }
-  
 
     const res = await fetch(endpoint, {
         body: JSON.stringify({
-            ...event,
+            timestamp: event.timestamp,
```

Comment:
Why this change?

Reply:
We are sending a tonne of stuff that we don't need which is confusing to maintain so I opted to just send the properties we actually support. Clearer and easier to reason on in the future